solidity/test/LiquidityProtection.js

```diff
@@ -144,7 +145,7 @@ describe('LiquidityProtection', () => {
         return new BN(transaction.gasPrice).mul(new BN(txResult.receipt.cumulativeGasUsed));
     };
 
-    const expectAlmostEqual = (amount1, amount2, maxError = '0.01') => {
+    const expectAlmostEqual = (amount1, amount2, maxError = 0.01) => {
```

Comment:
Using native floating-point numbers here is wrong.
We cast this string to `Decimal`.
When you cast `0.01` instead of `'0.01'`, you may get a different value.
This is also true for anywhere we call this function and pass this value explicitly (instead of leaving it as default).
Especially since we actually pass very small values in several different places.
The usage of strings for floating-point values is intentional.

Reply:
Gotcha. I'll revert.